index.html

```diff
@@ -0,0 +1,197 @@
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://kit.fontawesome.com/cdc6c51dd6.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Unicorn Con 2023 | Homepage</title>
+  </head>
+  <body>
+    <header class="header">
+      <ul class="nav justify-content-end nav-top bg-light d-none d-md-flex">
+          <li class="nav-item">
+            <a class="nav-link" aria-current="page" href="#"
+              ><i class="fa-brands fa-facebook"></i
+            ></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#"
+              ><i class="fa-brands fa-mastodon"></i
+            ></a>
+          </li>
+          <li class="nav-item"><a class="nav-link" href="#">Logout</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">English</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">My Page</a></li>
+      </ul>
+      <nav class="navbar navbar-expand-lg bg-dark navbar-custom sticky-top">
+        <div class="container-fluid">
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarToggler"
+            aria-controls="navbarToggler"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarToggler">
+            <a class="navbar-brand" href="index.html"
+              ><img
+                src="img/logo/kohai-logo-transparent-bg.png"
+                alt="logo"
+                class="logo"
+            /></a>
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link" href="pages/about.html">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#program">Program</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#guest">Guests</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#sponsor">Sponsors</a>
+              </li>
+            </ul>
+            <button type="button" class="btn btn-outline-danger">Claim your badge</button>
+          </div>
+        </div>
+      </nav>
```

Comment:
As shown in the design, the header is divided into a top dark bar and a bottom white bar. However, your design shows otherwise for the home desktop design. 

- [ ] Kindly make sure both designs follow the same pattern as shown below :point_down: .  
- [ ] Also check the update the margin between your logo and the lists of page links of the bottom white bar since it is different from the actual design. The lists of page links (About, Programs, Guests, etc)  should be aligned to the right.  


Your design

![Screenshot from 2023-01-12 07-35-26](https://user-images.githubusercontent.com/65233860/212011377-2c8b5c3d-22aa-4b91-b3f8-6ed3a73542c1.png)

The actual design

![Screenshot from 2023-01-12 07-34-46](https://user-images.githubusercontent.com/65233860/212011353-340ce4c3-63a1-4ef0-bbac-ada44a1c3c28.png)